packages/cli/src/__fixtures__/test-project/backstage.json

```
@@ -0,0 +1,3 @@
+{
+  "version": "1.7.0-next.0"
```

```suggestion
  "version": "1.0.0"
```

Because why have it be some weird version :p